notebooks/least_core_basic.ipynb

```diff
@@ -0,0 +1,811 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "995d4271",
+   "metadata": {},
+   "source": [
+    "# Least Core for Data Valuation\n",
+    "\n",
+    "This notebook introduces Least Core methods for the computation of data value using pyDVL.\n",
+    "\n",
+    "The Least Core is the solution to the following linear programming problem:\n",
+    "\n",
+    "$$\n",
+    "\\begin{array}{lll}\n",
+    "\\text{minimize} & \\displaystyle{e} & \\\\\n",
+    "\\text{subject to} & \\displaystyle\\sum_{i\\in N} x_{i} = v(N) & \\\\\n",
+    "& \\displaystyle\\sum_{i\\in S} x_{i} + e \\geq v(S) &, \\forall S \\subseteq N \\\\\n",
+    "\\end{array}\n",
+    "$$\n",
+    "\n",
```

Comment:
Can you add a few more words on how these equations are related to data valuation? Just an intuitive idea, if possible.

Reply:
Done. I also added a description of the difference between Shapley and the Core to both the notebook and the docs.

Reply:
there is a typo at the beginning of the notebook =)

Reply:
I fixed it. Thanks!